Lightning: add character set conversion configuration instructions (#6714)
pingcap/docs: tidb-lightning/tidb-lightning-configuration.md

#  - auto:    (default) automatically detects whether the schema is UTF-8 or
#             GB-18030. An error is reported if the encoding is neither.
#  - binary:  do not try to decode the schema files
character-set = "auto"

# Specifies the character set of the source data file. Lightning converts the source file from the specified character set to UTF-8 encoding when importing.
# Currently, this configuration only specifies the character set of the CSV files with the following options supported:
# - utf8mb4: Indicates that the source data file uses UTF-8 encoding.
# - GB18030: Indicates that the source data file uses the GB-18030 encoding.
# - GBK: The source data file uses GBK encoding (GBK encoding is an extension of the GB-2312 character set, also known as Code Page 936).
# - binary: Indicates that Lightning does not convert the encoding (by default).

> # Leaving this configuration blank means the default value "binary", that is to say, Lightning does not convert the encoding.

Reviewer: ```suggestion
# If left blank, the default value "binary" is used, that is to say, Lightning does not convert the encoding.
```
